lib/bilrost/bus.js

```diff
@@ -25,9 +25,8 @@ module.exports = function() {
   /**
    * Setup nconf to use (in-order):
    * 1. Environment variables
-   * 2. A file located at './config.test.json'
+   * 2. Config file that the app is using
```

Comment:
@faygao52 Have we checked that the tests still pass? I think the tests rely on that line you have deleted.

Reply:
@shirren All the test passed